src/styles/sidebar/components/tag-editor.scss

```diff
@@ -0,0 +1,45 @@
+@use "../../mixins/forms";
+@use "../../mixins/buttons";
+@use "../../variables" as var;
+
+.tag-editor {
+  &__input {
+    @include forms.form-input;
+    width: 100%;
+    &:focus {
+      @include forms.form-input-focus;
+    }
+  }
+
+  &__tag-list {
+    display: flex;
+    flex-wrap: wrap;
+  }
+
+  &__tag-item {
+    display: flex;
+    margin-right: 0.5em;
+    margin-bottom: 0.5em;
+  }
+
+  &__edit {
+    color: var.$grey-6;
+    background: var.$grey-1;
+    border: 1px solid var.$grey-3;
+    border-radius: 2px 0 0 2px;
+    border-right-width: 0;
+    padding: 0 0.5em;
+  }
+
+  &__delete {
+    @include buttons.action-button;
+    appearance: none;
+    border: 1px solid var.$grey-3;
+    border-radius: 0 2px 2px 0;
+    padding: 0 0.5em;
+    &:hover,
```

Comment:
This is the only button in the app with a border on hover—perhaps just background color/icon darker on hover?